target/linux/ipq40xx/image/mikrotik.mk

```diff
@@ -0,0 +1,18 @@
+define Device/mikrotik_nor
+	DEVICE_VENDOR := MikroTik
+	BLOCKSIZE := 64k
+	IMAGE_SIZE := 16128k
+	KERNEL_NAME := vmlinux
+	KERNEL := kernel-bin | append-dtb-elf
+	IMAGES = sysupgrade.bin
+	IMAGE/sysupgrade.bin := append-kernel | kernel2minor -s 1024 | pad-to $$$$(BLOCKSIZE) | \
+	append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
+endef
+
+define Device/mikrotik_routerboard-wap-g-5hacd2hnd
+  $(Device/mikrotik_nor)
```

Comment:
Please be consistent with indent. I think ipq40xx deviates from the other targets here by using tabs for indent inside Device block, so maybe we just use that consistently for this target now ...